simplified-app-simplye/src/main/assets/Accounts.json

```diff
@@ -2018,6 +2018,27 @@
         "supportsReservations": true,
         "supportsSimplyESync": true
     },
+    {
+        "catalogUrl": "http://qa.circulation.librarysimplified.org/QANYPL/",
+        "eulaUrl": "http://www.librarysimplified.org/EULA.html",
+        "id_numeric": 1254,
+        "id_uuid": "urn:uuid:00603c60-a261-4faf-a5eb-79107f3d7f9e",
+        "inProduction": false,
+        "licenseUrl": "http://www.librarysimplified.org/license.html",
+        "loginKeyboard": "Default",
+        "logo": null,
+        "mainColor": "red",
+        "name": "New York Public Library - QA Server",
+        "needsAuth": true,
+        "pinKeyboard": "Default",
+        "privacyUrl": "http://www.librarysimplified.org/privacypolicy.html",
+        "supportEmail": "ask@nypl.org",
+        "supportsBarcodeDisplay": false,
+        "supportsBarcodeScanner": false,
+        "supportsCardCreator": false,
+        "supportsReservations": true,
+        "supportsSimplyESync": true
+    },
```

Comment:
None of this data has changed, it's just been moved to be in alphabetical order. This was incorrectly put in the Qs -- I think I pasted it in manually thinking Q for "QA". Now that it's a real part of the library registry, it's generated by my script, and it's filed under N for "New York".